MovieApp/MovieApp/Model/Managers/FavoritesManager.swift

```diff
@@ -165,4 +175,24 @@ class FavoritesManager {
             }
         }
     }
+    
+    func addFavorite() {
+        if let movie = self.movieManager.movie {
+            self.addFavorite(movie: movie)
+        }
+    }
+    
+    func removeFavorite() {
+        if let movie = self.movieManager.movie {
+            self.removeFavorite(movie: movie)
+        }
+    }
+    
+    func isMovieFavorite(movieId: Int?) -> Bool {
```

Comment:
do you need this extra function instead of calling isMovieFavorite(id) directly?